README.md

```diff
@@ -1,7 +1,7 @@
-# PolymakeWrap
+# Polymake
 
-[![Build Status](https://travis-ci.org/sebasguts/PolymakeWrap.jl.svg?branch=master)](https://travis-ci.org/sebasguts/PolymakeWrap.jl)
+[![Build Status](https://travis-ci.org/sebasguts/Polymake.jl.svg?branch=master)](https://travis-ci.org/sebasguts/Polymake.jl)
```

Comment:
Shouldn't this point to `oscar-system`?

Reply:
That's true, but we also didn't setup CI so far.

Reply:
True. Which will be another delicate thing to do.